src/test/java/com/rarchives/ripme/tst/ripper/rippers/MulemaxRipperTest.java

```diff
@@ -0,0 +1,18 @@
+package com.rarchives.ripme.tst.ripper.rippers;
+
+import java.io.IOException;
+import java.net.URL;
+
+import com.rarchives.ripme.ripper.rippers.video.MulemaxRipper;
+import com.rarchives.ripme.utils.Utils;
+
+public class MulemaxRipperTest extends RippersTest {
+    public void testMulemaxVideo() throws IOException {
+        // This test fails on the CI so we skip it unless running locally
```

Comment:
I haven't; I wasn't thinking about it when I copied the SpankBangRipperTest after your changes and didn't think to revert that section to try it on Travis.

I can do that and re-push if you'd like.

Reply:
> I can do that and re-push if you'd like.

Please do. I'd rather not have tests disabled if they don't have to be

Reply:
Done.

Reply:
Looks like the test is failing with "Failed to download a single file from https://cdn901.mulemax.com/media/videos/h264/1720_480p.mp4". No idea what's going on with that, it doesn't seem to effect any of the other rippers

Reply:
I think it's still a Travis issue - I keep running `mvn test` locally, and it's not failing on the ones I've added.  I make sure to fetch the upstream and merge before I do; and I've been able to build clean and rip from the site with the new ripper, so I'm really at a loss.

Reply:
> I think it's still a Travis issue

I think it might be a ripme/linux issue. Running the tests locally they seem to fail very rarely (Less than 5% of the time). What I think is happening is rip is checking for the downloaded file before the file actually gets written to disk. Tbh the entire video ripper code is a bit of a mess and needs to be replaced. If you write anymore rippers I'd recommend using `AbstractSingleFileRipper`. You can see an example for it in the "GfycatporntubeRipper"

Anyhow readd the Travis CI Unit Test bypass and I'll merge this

Reply:
Ok, so I re-added the bypass.

But apparently my efforts to continue building Rippers in a separate branch has back fired and those commits were included here.  I apparently don't understand Git.

Sorry about that.

Reply:
It also looks like the StickyXXXRipperTest is failing, add the Travis CI Unit Test bypass to that and I'll merge